### PR TITLE
South Africa_Samuel_Kau_Homework_errors_fixed_week2

### PR DESCRIPTION
**Pull request by Samuelkau**
# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

**Comment by JarrodBen** [comment]
Hey there Samuel hope you're having a great day, your question 6 and question 15 still needs a fixing and your question 4 isn't exactly wrong, but it's just a typo, it's usually written as Hypertext Markup Language (just a small error), but all in all the rest of it seems to be correct from what I can see.

**Comment by JarrodBen** [comment]
I have reviewed your work thoroughly before I gave any comments on where there might be errors.

File: homework.md [hunk 1]
```diff
@@ -4,89 +4,89 @@
...
-
 }
 ```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+JavaScript is a text-based programming language used both on the client-side and server-side that allows you to make web pages interactive
```
Comment (Ntshembo-Hlongwane1) [inline homework.md:46]: So here on your answer you were on the right track till you then included, "What JavaScript does" which is out the scope of the question plus where you even deviated way more far from your answer is when you said, 
>  allows you to make web pages interactive

# Key on this question:
- Is for you to identify what JavaScript is, is it an Object Oriented Programming Language, Interpreted Programing Language, Compiled Programming Language etc and not to explain what it does.

**Comment by russmahn** [comment]
Hi Samuel regarding question 16 could you remove the HTML

**Comment by JarrodBen** [comment]
Hey there Samuel hope you're having a great day, your question 6 and question 15 still needs a fixing and your question 4 isn't exactly wrong, but it's just a typo, it's usually written as Hypertext Markup Language (just a small error), but all in all the rest of it seems to be correct from what I can see. This is an //old comment//

**Pull request closed without merging**